web-service-consumer/1.10/modules/ROOT/pages/web-service-consumer-reference.adoc

```diff
@@ -63,7 +63,7 @@ ConnectionProvider that returns instances of WscSoapClient.
 | Decrypt Security Strategy a| <<wss-decrypt-security-strategy>> |  A decrypt WSS configuration |  | 
 | Encrypt Security Strategy a| <<WssEncryptSecurityStrategyAdapter>> |  an encrypt WSS configuration |  | 
 | Incoming Timestamp Security Strategy a| <<WssIncomingTimestampSecurityStrategyAdapter>> |  A timestamp verification WSS configuration |  | 
-| Must Understand a| Boolean |  Whether mustUnderstand attribute in wsse:Security header is true or false.  Defaults to true TODO (W-11800462): For the next major (v2.0.0) the default value should be false to match the SOAP protocol standard. |  true | 
+| Must Understand a| Boolean |  Determines whether the *Must understand* attribute in the Web Service Security header is `true` or `false`. Defaults to `true`. |  true | 
```

Comment:
```suggestion
| Must Understand a| Boolean |  Determines whether the *Must understand* attribute in the Web Service Security header is `true` or `false`. |  true | 
```

Reply:
If the next column defines the default value, there's no need to explain that "Must understand" defaults to "true".